paywall/src/unlock.js/module.ts

```diff
@@ -0,0 +1,68 @@
+import startup from './startup'
+import '../static/iframe.css'
+import { UnlockWindowNoProtocolYet } from '../windowTypes'
+import StartupConstants from './startupTypes'
+
+declare const process: {
+  env: {
+    UNLOCK_ENV: 'dev' | 'test' | 'staging' | 'prod'
+    DEBUG: any
+    PAYWALL_URL: string
+    USER_IFRAME_URL: string
+  }
+}
+
+const constants: { [key: string]: StartupConstants } = {
+  dev: {
+    network: 1984,
+    debug: process.env.DEBUG,
+    paywallUrl: process.env.PAYWALL_URL || 'http://localhost:3001',
+    accountsUrl: process.env.USER_IFRAME_URL || 'http://localhost:3000/account',
+  },
+  test: {
+    network: 1984,
+    debug: process.env.DEBUG,
+    paywallUrl: process.env.PAYWALL_URL || 'http://localhost:3001',
+    accountsUrl: process.env.USER_IFRAME_URL || 'http://localhost:3000/account',
+  },
+  staging: {
+    network: 4,
+    debug: process.env.DEBUG,
+    paywallUrl: process.env.PAYWALL_URL,
+    accountsUrl: process.env.USER_IFRAME_URL,
+  },
+  prod: {
+    network: 1,
+    debug: process.env.DEBUG,
+    paywallUrl: process.env.PAYWALL_URL,
+    accountsUrl: process.env.USER_IFRAME_URL,
+  },
+}
+
+// TODO: make this more sophisticated? Would be nice to be able to pass config
+// as a param instead of relying on a window variable. For now this is just a
+// slight modification of `index.ts`
+export function init() {
```

Comment:
This is simple enough, it just wraps what the script would ordinarily do automatically with a function that gets exported.